Dockerfile

```diff
@@ -67,7 +67,7 @@ RUN pg_ctl start ; \
 USER root
 RUN mkdir /opt/odoo-addons
 RUN mkdir /opt/odoo-addons/addons
-RUN chmod -R 777 /opt/odoo-addons/
+RUN chown -R odoo /opt/odoo-addons/
```

Comment:
Please do not include in the pull request commits that undo mistakes from earlier commits in the series.  Instead, use rebase as necessary to fix the original commits.